medusa/notifiers/telegram.py

```diff
@@ -68,7 +68,7 @@ def _send_telegram_msg(self, title, msg, user_id=None, api_key=None):
             success = True
         except RequestException as error:
             message = 'Unknown IO error: %s' % error
-            if hasattr(error, 'response'):
+            if hasattr(error, 'response') and error.response is not None:
```

Comment:
IMO you could also do this:
```py
if getattr(error, 'response', None):
```

Reply:
I opted to keep `hasattr` because we are just interested in knowing if `error` has the `response` here and don't want to actually use it directly. It just reads better IMO.